tests/sles4sap/saptune/mr_test.pm

```diff
@@ -48,10 +48,10 @@ sub setup {
     # Disable packagekit
     quit_packagekit;
     # Install saptune
-    zypper_call "-n in saptune";
+    zypper_call "in saptune";
+    zypper_call "in sapconf";
```

Comment:
Why not `zypper_call "in saptune sapconf"`?

Reply:
It is OK too.